Tests/Unit Tests/Conditions/PostcodeConditionTests.swift

```diff
@@ -16,48 +16,187 @@ final class PostcodeConditionTests: XCTestCase {
     
     // MARK: - Properties
     
-    let condition       = PostcodeCondition(country: .UnitedKingdom)
+    let conditionGBR        = PostcodeCondition(country: .UnitedKingdom)
+    let conditionSWE        = PostcodeCondition(country: .Sweden)
+    let conditionTUR        = PostcodeCondition(country: .Turkey)
     
     
-    // MARK: - Test Success
+    // MARK: - GBR - Test Success
```

Comment:
This is a good start. I think I'd like to refactor these tests in a future PR to help remove the redundancy. 🤔